app.py

```diff
@@ -2,9 +2,6 @@
 from flask import Flask, request, jsonify, json, session
```

Comment:
we could also delete `request`, `json`, and `session` since they're not being used in this file

Reply:
Is it ok if we keep them? I think they are handy, and we could definitely use them in the future.

Reply:
MHM!

Reply:
Thanks. If we work on our future iterations, we will definitely need these.